test/integration/src/test/java/io/pravega/test/integration/KeyValueTableTest.java

```diff
@@ -96,6 +160,10 @@ public void setup() throws Exception {
         SocketConnectionFactoryImpl connectionFactory = new SocketConnectionFactoryImpl(clientConfig);
         this.connectionPool = new ConnectionPoolImpl(clientConfig, connectionFactory);
         this.keyValueTableFactory = new KeyValueTableFactoryImpl(SCOPE, this.controller, this.connectionPool);
+
+        //5. setting up controller service instance
+        this.controllerService = new ControllerServiceImpl(controllerWrapper.getControllerService(),
```

Comment:
The controller is already set up. Remove this .

Reply:
Supporting the above comment to make use of ControllerServiceImp instance.